test/Where.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from "chai";
+import { argv0 } from "process";
+import { DangerLinq } from "../src";
+import { DangerLinqFunction } from "../src/DangerLinqFunction";
+
+describe("DangerLinqFunction.Map", () => {
+    it("Should be a function", async () => {
+      const Where = await DangerLinq(DangerLinqFunction.Map);
+      expect(Where).to.be.a("function");
+    });
+
+describe('Array#prototype#Where', function() {
+  const input = [1, 2, 3, 4, 5, 6];
+
+  it('Should return only even numbers', function() {
+    const expected = [2, 4, 6];
+    const actual = input.Where(x => x % 2 === 0);
```

Comment:
DangerLinq doesn't add the function to the array prototype, the code won't run like this (unlike https://github.com/ShittyLabs/ShittyLINQ.js). The test should look something like:
```javascript
const Where = await DangerLinq(DangerLinqFunction.Where);
const actual = where(input, x => x % 2 === 0);

expect(actual).to.eql(expected);
```